apps/api/src/utils/format.ts

```
@@ -0,0 +1,280 @@
+export function sanitizeInput(input: string): string {
+  return input
+    .replace(/[<>]/g, '') // 移除HTML标签
+    .replace(/[\u0000-\u001F\u007F-\u009F]/g, '') // 移除控制字符
+    .replace(/&/g, '&amp;') // 转义特殊字符
+    .replace(/"/g, '&quot;')
+    .replace(/'/g, '&#39;')
+    .trim() // 移除首尾空格
+}
+
+// 日期格式化
+export function formatDate(date: Date, format: string = 'YYYY-MM-DD HH:mm:ss'): string {
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)